ably-iosTests/ARTRealtimeChannelTest.m

```diff
@@ -39,13 +39,13 @@ - (void)tearDown {
     // Put teardown code here. This method is called after the invocation of each test method in the class.
     if (_realtime) {
         [_realtime removeAllChannels];
-        [_realtime.eventEmitter removeEvents];
+        [_realtime resetEventEmitter];
         [_realtime close];
     }
     _realtime = nil;
     if (_realtime2) {
         [_realtime2 removeAllChannels];
-        [_realtime2.eventEmitter removeEvents];
+        [_realtime2 resetEventEmitter];
```

Comment:
`off` is for listeners to unsubscribe themselves, isn't it? This is a (private) method that removes all listeners from the emitter's side. It's just a helper for tests, to clean up and avoid calling callbacks when closing the channels once the test is done.